cmd/seid/cmd/root.go

```diff
@@ -250,6 +251,9 @@ func newApp(
 		panic(err)
 	}
 
+	wasmGasRegisterConfig := wasmkeeper.DefaultGasRegisterConfig()
+	wasmGasRegisterConfig.GasMultiplier = 8750000
```

Comment:
can we add a comment explaining what the default is and why we chose this number? 2-3 sentences shoudl be sufficient

Reply:
also nit - could we do `8_750_000`

Reply:
yup, wil fix